makefile

```
@@ -1,25 +1,48 @@
 # Default target
 .PHONY: all
-all: openttd-sans openttd-serif openttd-small openttd-mono
+all: ttf openttd-sans openttd-serif openttd-small openttd-mono
```

```suggestion
all: openttd-sans openttd-serif openttd-small openttd-mono
```

ttf doesn't need to be a dependency of all

---

Made equivalent changes to preview directory and rule dependencies too.